tests/features/site-team.feature

```diff
@@ -32,9 +37,14 @@ Feature: Managing a site's team
 
   @vcr site-team-role.yml
   Scenario: Changing a team member's role
-    When I run "terminus site:team:role [[test_site_name]] [[other_user]] admin"
+    When I run "terminus site:team:role [[test_site_name]] [[other_user]] developer"
```

Comment:
This wasn't validated before, and the admin role became invalid between when the test was written and now.